### DISCUSSION TITLE
The app is not working

### DISCUSSION THREAD

**Discussion by IlSuperGallo** [nigification/Flowgorithm-Mac-OSX#17] [Q&A] [answered]
On my Macbook air with M1 and updated to Ventura 13.3.1 the app is not working; when i try to open it nothing shows up, not even a window.

**Comment by jostasik** [comment, marked as answer]
This has been fixed with the latest release. If you're still facing this issue,  remove the current app from your computer and then replace it with this: https://github.com/jostasik/Flowgorithm-macOS/releases/tag/v3.4.2
